components/main/main.styled.ts

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+import Breakpoints from "../../constans/breakpoints";
+
+export const Content = styled.div`
+  width: 100%;
+  overflow: auto;
+  min-height: calc(100vh - 164px);
```

Comment:
what is 164px? looks like a magic number. Seems like it is header + footer height - then it makes sense to import their heights here and reuse them to calculate this value.
Let imagine that header height is increased and not it is 100px instead of 60px... In that case this logic will be broken and you will have ugly scrollbar

Reply:
So you moved it to the variable, which is fine. But did you check this situation, described in comment:
"Let imagine that header height is increased and now it is 100px instead of 60px... In that case this logic will be broken and you will have ugly scrollbar"

Do you understand what was meant here: "makes sense to import their heights here and reuse them to calculate this value"?